docs/source/users.rst

```diff
@@ -3,8 +3,28 @@ User Management
 
 This section covers the customer owner, adding and removing users, email on-boarding and user preferences.
 
+Definitions
+-----------
+Project Koku uses terminology in certain ways to describe key concepts. These are definitions of the term based on how they work within the context of the Koku and Masu applications.
+
+.. _customer:
+
+Customer: An organization or entity that uses Project Koku for cost management analysis.
+
+.. _customer owner:
+
+Customer Owner: A user_ that is responsible for management of a customer_. Customer owners have administrative privileges over Customer data.
+
+.. _provider:
+
+Provider: A cloud resource provider or cloud data provider. An entity that produces cost and resource usage data.
+
+.. _user:
+
+User - A user of the Project Koku application. Users map to an individual person or login.
+
 Customer Owner
```

Comment:
Can be removed.

Reply:
I'm going to leave it, just so it's clear that User and Customer are not the same thing.